zcash_client_backend/src/data_api.rs

```diff
@@ -466,6 +466,32 @@ impl SentTransactionOutput {
     }
 }
 
+#[derive(Clone, Debug)]
+pub struct AccountBirthday {
+    height: BlockHeight,
+    sapling_frontier: Frontier<Node, NOTE_COMMITMENT_TREE_DEPTH>,
+}
+
+impl AccountBirthday {
+    pub fn from_parts(
+        height: BlockHeight,
+        sapling_frontier: Frontier<Node, NOTE_COMMITMENT_TREE_DEPTH>,
```

Comment:
As discussed in pairing, this is required so that it's possible to guarantee that the note commitment tree has retained enough nodes to make a witness.